parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -696,4 +704,32 @@ disableSSH() {
     systemctlDisableAndStop ssh || exit $ERR_DISABLE_SSH
 }
 
+configCredentialProvider() {
+    CREDENTIAL_PROVIDER_CONFIG_FILE=/var/lib/kubelet/credential-provider-config.yaml
+    mkdir -p "$(dirname "${CREDENTIAL_PROVIDER_CONFIG_FILE}")"
+    touch "${CREDENTIAL_PROVIDER_CONFIG_FILE}"
+    tee "${CREDENTIAL_PROVIDER_CONFIG_FILE}" > /dev/null <<EOF
```

Comment:
nit: just use `cat` instead of tee'ing and piping output to the null device

Reply:
tee is the common tool used in case to write files, so I it's good to keep it as it is.